### PR TITLE
Adding the option to disable the docker save in save scripts

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/30062

### CODE REVIEW

File: pkg/image/export/main.go [hunk 1]
```diff
@@ -513,7 +513,8 @@ images="rancher-images.tar.gz"
 usage () {
     echo "USAGE: $0 [--image-list rancher-images.txt] [--images rancher-images.tar.gz]"
     echo "  [-l|--image-list path] text file with list of images; one image per line."
-    echo "  [-i|--images path] tar.gz generated by docker save."
+		echo "  [-i|--images path] tar.gz generated by docker save."
```
Comment (leodotcloud) [inline pkg/image/export/main.go:516]: Incorrect use of tabs/spaces. Please correct.

File: pkg/image/export/main.go [hunk 2]
```diff
@@ -513,7 +513,8 @@ images="rancher-images.tar.gz"
...
+		echo "  [-d|--disable-save] disable docker save and only pull images."
```
Comment (leodotcloud) [inline pkg/image/export/main.go:517]: Generally `-d` is used as debug enable, it's better to not use this short form. Also, for the long form, it's better to use a more descriptive arg. Suggest: `[--disable-docker-save]`.

File: pkg/image/export/main.go [hunk 3]
```diff
@@ -525,6 +526,10 @@ while [[ $# -gt 0 ]]; do
         images="$2"
         shift # past argument
         shift # past value
+        ;;
+				-d|--disable-save)
```
Comment (leodotcloud) [inline pkg/image/export/main.go:530]: Same as above. Issue with tabs/spaces.

**Comment by leodotcloud** [comment]
@mattmattox Created the draft PR on your behalf. Have a few review comments, can you please take care?

File: pkg/image/export/main.go (hunk 1 above)
Comment (mattmattox) [reply pkg/image/export/main.go]: This is fixed (I had to go into Atom and fix Soft Tabs.

File: pkg/image/export/main.go (hunk 2 above)
Comment (mattmattox) [reply pkg/image/export/main.go]: No problem, I removed `-d` and swapped it as requested.

File: pkg/image/export/main.go (hunk 3 above)
Comment (mattmattox) [reply pkg/image/export/main.go]: Fixed

**Comment by leodotcloud** [comment]
Closed this in favor of https://github.com/rancher/rancher/pull/30066

**Pull request closed without merging**